uktrade/data-hub-frontend
# Change OMIS UK region filter to typeahead

## Description of change

The UK Region filter on the Orders (OMIS) page is now a _typeahead_.

## Test instructions

1. Go to the Orders section.
2. On the very bottom of the filters sidebar, you should see that the UK region filter is now a _typeahead_
3. Focus the field, type the name of your region of choice, select it by either clicking on it or hit enter
4. You should see that the search results reflect the filter's setting

## Screenshots
### Before

<img width="521" alt="Screen Shot 2020-02-06 at 11 27 56 AM" src="https://user-images.githubusercontent.com/2333157/73933594-afec7f80-48d4-11ea-929f-da9677ddb35f.png">

### After
<img width="493" alt="Screen Shot 2020-02-06 at 11 26 13 AM" src="https://user-images.githubusercontent.com/2333157/73933597-b11dac80-48d4-11ea-8322-a94116abd9d3.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
